Source/Data/Remotes/DataRequest+responseDecodableAtKeyPath.swift

```diff
@@ -0,0 +1,40 @@
+import Alamofire
+import Foundation
+
+extension DataRequest {
+
+    enum KeyPath {
+        case data
+        case none
+    }
+
+    @discardableResult
+    func responseDecodableAtKeyPath<T: Codable>(
+        of type: T.Type = T.self,
+        keyPath: KeyPath,
```

Comment:
I've tried to get this to work with a [referencewritablekeypath](https://developer.apple.com/documentation/swift/referencewritablekeypath) but I did not get it to work. So defined a custom `KeyPath` enum witch we can switch over.